includes/Modules/Tag_Manager.php

```diff
@@ -427,6 +427,30 @@ protected function get_datapoint_services() {
 		);
 	}
 
+	/**
+	 * Gets map of datapoint to definition data for each.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array Map of datapoints to their definitions.
+	 */
+	protected function get_datapoint_definitions() {
+		$map = parent::get_datapoint_definitions();
+
+		// TODO: remove this once datapoint exists.
```

Comment:
See above, eventually all datapoints should be in here anyway, no more `get_datapoint_services()`.